### ISSUE TITLE
Create an official Jupyter Mastodon account

### ISSUE THREAD

**Issue by choldgraf** [jupyter/governance#146]
Many open source communities and projects are migrating to Mastodon right now. As part of that process, several organizations are defining "official" accounts in a Mastodon server, so that others know where is the source of truth for communications from that organization.

Several examples of "official" accounts in Mastodon are provided by @krassowski here:

- https://discourse.jupyter.org/t/official-mastodon-account-s/16688/8?u=choldgraf

Given the high-visibility nature of Jupyter, I think it's important that we provide this source of truth in Mastodon to avoid confusion about where our communications are coming from.

However, there's an open question about **which Mastodon server Jupyter should use**, there's been some discussion of this in the discourse thread:

- https://discourse.jupyter.org/t/official-mastodon-account-s/16688

Given that we don't yet know if there will be an "official" Python projects server, and given that it is relatively easy to transition to new servers if you wish, I propose we take the following path:

### Proposal

- Wait for three weeks to see if an "official" Python or Scientific Python server arises that we can use.
- After **December 19th**:
- If it does, then create a `projectjupyter@<this-new-server.org>` account
- If not, then create an official Jupyter social media account on `fosstodon.org` (this is the server many other major projects have used)
- Document it as our official Jupyter account, and add a `rel="me"` link to our home page to this account.

### Current state as of Dec 21 2022

There has been significant discussion and some actions on this, so I am summarizing the current state of things below:

- We discussed this topic in the Discourse thread linked above in Nov
- This issue was opened ~3 weeks ago with a specific proposal for using the `fosstodon.org` server
- Initially the EC suggested waiting on making an account until the new EC had been formed, had created a social media team, and delegated this decision to that team.
- The EC reconsidered this decision in their office hours meeting and agreed to allow creating an account on `fosstodon` about two days ago. https://github.com/jupyter/governance/issues/146#issuecomment-1358118062
- This account was created and linked from jupyter.org, though not announced from an official jupyter account: https://github.com/jupyter/jupyter.github.io/pull/710#issuecomment-1359020491
- Yesterday some folks expressed concern that `fosstodon.org` is an english-only server, a point that had not been discussed before, and recommended moving the account immediately or shutting it down.
- @choldgraf has deleted his twitter/mastodon posts about this, given the uncertainty of whether we'll use this moving forward.
- There's a PR open here to revert the jupyter.org changes: https://github.com/jupyter/jupyter.github.io/pull/711
- We are now awaiting the EC's decision on how to move forward.

**Comment by manics** [comment]
Is Jupyter still a Python project, or has it grown beyond that? I'm mostly involved with the Python components so I can't really tell!

**Comment by choldgraf** [comment]
My feeling is that technically, it is much more than python. But culturally, it overlaps more with the scientific python community much more than any other major open source community.

That said, I just noted the scientific python server because I know those conversations are happening. But I am happy to go with a different proposal in the name of moving forward. It's more important that there's an official account _somewhere_ than if it exists on the perfect server.

**Comment by krassowski** [comment]
My feeling is that it has a reach and aspires much beyond Python. Since both Python and R foundation are just on Fosstodon this is what I believe is the best course of action as of today. Julia has its own server.

Agree that any server is better than no presence at all, but for the same reason I believe Fosstodon (or floss.social) now is better than waiting (accounts can be moved around easily).

**Comment by minrk** [comment]
Following Python itself (and other projects like pandas) to Fosstodon and adding the rel=me now makes sense to me. Migrating to a 'scientific python' instance, if one is launched, could make sense, too. I've already migrated servers once, and it's no big deal.

**Comment by jweill-aws** [comment]
We discussed this at the governance meeting this morning. We are still in the process of electing the new Executive Council (voting closes on Sunday December 4). The new EC should work with NumFOCUS to coordinate a social media strategy, wither on Fosstodon or another instance. We will follow up on this at a later EC meeting. Thank you all for your engagement!

**Comment by Carreau** [comment]
When creating the Jupyter account, please also create an IPythonDev one, and add credentials to the onepassword.

**Comment by choldgraf** [comment]
@jweill-aws to clarify your statement, is this the current state of things?

_The governance meeting team concluded that **nobody should create this account until the Executive Council has created a strategy and then defines a policy about Mastodon accounts**._

If so I will hold off on creating anything. Also just a note that there is now at least one account that is called `jupyter` on Mastodon.

![image](https://user-images.githubusercontent.com/1839645/205887932-fd926c75-0b72-4ed2-8d19-5c46b3d7643f.png)

**Comment by jweill-aws** [comment]
@choldgraf Yes, we just had our EC election, and we are in the process of spinning up the EC.

The EC's office hours are scheduled for Mondays at 10:00 Pacific (18:00 UTC, currently), and I encourage you to bring this up at that time, if you're available.

**Comment by choldgraf** [comment]
I'm usually not available at that time, as it's 7pm in France and I'm on baby duty then. Anyway, I don't think there's any information that I'd share in that meeting that isn't already documented in this issue.

I'll leave this open so that the executive council can decide what to do with it. But feel free to close if you wish.

**Comment by jweill-aws** [comment]
I usually attend the office hours, so I can continue to drive this.

On Dec 6, 2022, at 12:13, Chris Holdgraf ***@***.***> wrote:

﻿

I'm usually not available at that time, as it's 7pm in France and I'm on baby duty then. Anyway, I don't think there's any information that I'd share in that meeting that isn't already documented in this issue.

I'll leave this open so that the executive council can decide what to do with it. But feel free to close if you wish.

**Comment by Carreau** [comment]
> The governance meeting team concluded that nobody should create this account until the Executive Council has been created, then created a social media strategy, and then defines a policy about Mastodon accounts.

As a note, there is a difference between creating/using and registering the name. 

Is the decision definitely about registering/reserving the name, or more about using the media ? We already have a number of other social media accounts (and domains)  that have been registered but do not display any jupyter-related name/url/avatar but are under our control just in case.

**Comment by choldgraf** [comment]
Agreed - in my opinion jupyter should define a "source of truth" handle, even if it intends to never use it, just so that it's clear what is official and what is not. This would be a very low effort action.

**Comment by krassowski** [comment]
What is the status here? A ship has somewhat already sailed. Do we really need a months-long process to setup a handle?
- if we wanted to give users heads up from Twitter account that there is an official account on Mastodon it is too late since Twitter started blocking links to Mastodon (yesterday?)
   - this was important consideration for setting up account early since large fraction of followers are passive (unless they are hard-core users/developers they will likely not remember that they had followed Jupyter on the other platform) and needed a nudge
- there is still time to to set up an account and there is still benefit in doing it earlier than later (users with interest in specific news sources will actively seek them just after setting up account and less so once well settled and happy with their newsfeed, and there are still swathes of users migrating)

**Comment by jweill-aws** [comment]
@krassowski It's been 3 weeks since this issue was opened, not months, and the request also coincided with the transition of governance to the new Executive Council. As Twitter's capricious overlord continues to make decisions that adversely affect the platform's viability, I agree that setting up a presence on Mastodon would benefit the project. I intend to join the Jupyter EC office hours at 10:00 US/Pacific today to bring this topic up again.

**Comment by jweill-aws** [comment]
I have submitted a request for an official Jupyter account on FOSStodon, using an official Jupyter e-mail account. Once this is created, I can update Jupyter's web site to link to our Jupyter accounts so that they show up using Mastodon's green-check verification. Note that I'll be away for the holidays. Also note that FOSStodon's action button is "Add to Waitlist," not "Register Account," so our account requires approval to be created. Thank you for your patience.

**Comment by jweill-aws** [comment]
https://fosstodon.org/@Jupyter is now online. I can update the related PR to only add `<link>` tags, but not link to this account, since it has only a basic profile at the moment and is not being actively used yet.

**Comment by jweill-aws** [comment]
Additional to-dos (I'm OK if we open separate issues for them)

- [ ] Use an app like [fedifinder](https://www.fedifinder.com/) to scan all of `@ProjectJupyter`'s Twitter follows for Mastodon accounts, and then follow them on Mastodon
- [ ] Update `@ProjectJupyter`'s Twitter bio to mention `@Jupyter@fosstodon.org` (this is [no longer a bannable offense](https://news.yahoo.com/twitter-suddenly-reverses-course-policy-154000080.html) on Twitter)
- [ ] Come up with a strategy to cross-post content on Twitter and Mastodon (probably requires more discussion)

**Comment by jweill-aws** [comment]
@Carreau I have requested `@IPythonDev` on Fosstodon; will add credentials to the 1Password.

**Comment by choldgraf** [comment]
Thanks @jweill-aws for doing this work 👍

**Comment by jweill-aws** [comment]
We are "verified"!

![Screen shot of Jupyter's Fosstodon profile showing green checkmark for website](https://user-images.githubusercontent.com/93281816/208693146-ed4686ac-2f3b-41c1-8dfc-a06e439c676c.png)

**Comment by SylvainCorlay** [comment]
<img width="516" alt="Screenshot 2022-12-20 at 19 04 57" src="https://user-images.githubusercontent.com/2397974/208735685-4475de9e-1fb8-4402-8d61-6d398215841b.png">

Fosstodon prohibits posts in other languages than English, which I think is not great. In the past, we have promoted blog posts in other languages on social media.

**Comment by jweill-aws** [comment]
I agree that the English-only rule is not ideal for us. One advantage of Mastodon is that we can move our account to another server if this rule becomes problematic. Alternatively, if Fosstodon's moderation team expands to be more multilingual, they may be more receptive to posts in other languages.

**Comment by krassowski** [comment]
Given the tagline includes "please open issues on GitHub" I would suggest we add two links to the two GitHub repositories  which we actually do monitor (in weekly triage meetings):
- https://github.com/jupyterlab/jupyterlab
- https://github.com/jupyter/notebook

This is because we see users opening issues in irrelevant repositories such us jupyterlab-git (because how SEO works when you search for "jupyterlab github") or the documentation repository (https://github.com/jupyter/jupyter.github.io/issues, probably because docs build often link back to source github).

For context, Mastodon allows up to 4 links and this one still empty. The list of links could be revised later, especially if an assistant github template was created (we could use template selector to link to multiple repositories).

**Comment by jweill-aws** [comment]
@krassowski Perhaps a link to the [community page](https://jupyter.org/community) might be better? There are a lot of reasons why a person/org might need to contact Jupyter, and a GitHub issue might not work well for everyone.

**Comment by krassowski** [comment]
The community page is already indirectly linked via the main website. My concerns is only that the tagline points to "github" without providing a link and users may end up lost.

**Comment by SylvainCorlay** [comment]
> > Fosstodon prohibits posts in other languages than English
>
> I agree that the English-only rule is not ideal for us.

I am kinda bummed by this pick. So it is completely forbidden for ProjectJupyter to post content in my mother tongue. There is a lot of content about/on Jupyter in French (and probably in other languages) that we should be able to promote. I think we should take down the fosstodon account and move to another server.

**Comment by jweill-aws** [comment]
@SylvainCorlay I looked at the last two years' worth of `@ProjectJupyter` tweets, including replies, and found that they were all written in English (although I did not look at the links from those tweets). The most recent non-English Tweet I could find was from August 12, 2020, in Turkish. (That tweet was part of a small thread about localization, and consisted of the same content in English, French, Spanish, Portuguese, Finnish, and Turkish, so it would likely be consistent with Fosstodon's moderation policy.)

In addition, despite its stated policy, Fosstodon is not wholly in English; [this post from earlier today](https://fosstodon.org/@grndstt/109547471705494226) is in French, for example. The rule is meant to facilitate moderating potentially controversial or disputed content, which I don't expect Project Jupyter to post.

**Comment by SylvainCorlay** [comment]
"Do not post in languages other than English" is pretty clear. I think that forbidding explicitely other languages does not work well with our values with respect to inclusivity.

**Comment by SylvainCorlay** [comment]
I think we should resolve this situation as soon as possible. If picking a server is too complicated, I would be in favor of deleting this account.

**Comment by choldgraf** [comment]
For what it's worth, I'm +1 on moving to whatever server. As @jweill-aws notes i don't believe this is a technically difficult thing to do and we don't pay much of a penalty for doing it. Maybe the other servers in the list that @krassowski shared in the discourse post don't have language restrictions?

**Comment by SylvainCorlay** [comment]
I am in favor of just deleting this account right away. Creating this and advertising it as the official accountwent against the decision that this should be done after the Executive Council has been created:

> The governance meeting team concluded that nobody should create this account until the Executive Council has been created, then created a social media strategy, and then defines a policy about Mastodon accounts.

**Comment by choldgraf** [comment]
I'm fine just deleting it too - my impression from [this comment](https://github.com/jupyter/governance/issues/146#issuecomment-1358118062) was that this had been discussed in the office hours of the EC, thus triggering the action to sign up for an account, but I think that I did not interpret that correctly.

**Comment by jweill-aws** [comment]
I have discussed this with three of the incoming EC's members, and the consensus was that we should proceed and that Fosstodon was a good place to situate our account. I am also going to be out of office and incommunicado December 21–28.

If you want to remove the "official" designation from our Jupyter account, I'm OK with that; a pull request to back out https://github.com/jupyter/jupyter.github.io/pull/710 would accomplish that goal.

**Comment by SylvainCorlay** [comment]
> I have discussed this with three of the incoming EC's members, and the consensus was that we should proceed and that Fosstodon was a good place to situate our account. I am also going to be out of office and incommunicado December 21–28.


Thanks for the clarification. I will reach out to the EC about this.

**Comment by choldgraf** [comment]
Thanks for clarifying @jweill-aws - in that case I support whatever decision the EC wants to make about next steps. I've opened a PR to revert this in case that is the decision for the website, but I do not have the access to modify the hachyderm account itself.

**Comment by SylvainCorlay** [comment]
I still think we need to delete this account asap. This definitely sends a wrong signal.

**Comment by Carreau** [comment]
I just want to note:

```
Exceptions
There are exceptions to this rule. All we really ask is the initial Toot is posted be in English. By default, people don’t see replies in their local timelines. If you end up speaking German in DM, or in replies, so what? As long as that conversation doesn’t breach our CoC we’re all good.
```

from https://hub.fosstodon.org/why-fosstodon-is-english-only/, 

So If we tweet in english and have translations or reply in french, it does not seem to be an issue.


I don't know if it's mitigating or not.

I would prefer to not delete in case we decide to stay there, we might not be able to recreate if deleted.

**Comment by Carreau** [comment]
I just want to note:

```
Exceptions
There are exceptions to this rule. All we really ask is the initial Toot is posted be in English. By default, people don’t see replies in their local timelines. If you end up speaking German in DM, or in replies, so what? As long as that conversation doesn’t breach our CoC we’re all good.
```

from https://hub.fosstodon.org/why-fosstodon-is-english-only/, 

So If we tweet in english and have translations or reply in french, it does not seem to be an issue.


I don't know if it's mitigating or not.

I would prefer to not delete in case we decide to stay there, we might not be able to recreate if deleted.

**Comment by SylvainCorlay** [comment]
As much as I am in favor of using the most used language for international audiences (I run the PyData Paris meetup, which is one of the very few English language meetups in Paris), I think that having a policy forbidding people to use the mother tongue on the project account is plain wrong. I initially thought this was an oversight, but I cannot stand by a decision condoning this.

**Comment by fperez** [comment]
Speaking as a member of the EC, but not on its behalf here - this happened as the finalization of the EC and the holidays overlapped, so it's tricky to get everyone's voice in at the same time.

I wasn't aware of the language limitations with fosstodon. English isn't my native language and one of my priorities moving forward with Jupyter and more generally open source/open science, is to make a strong push in Latin America for more participation and engagement in those regions (that happens to be where my networks/connections lie, it's equally important to push to other regions beyond North America/Europe).

So I absolutely support that we find a solution for our social media that is language-inclusive. 

But I also don't think deleting the fosstodon account is the right choice. Fosstodon is shaping as a key hub for OSS projects/people, and as such we should have _a_ presence on it. Furthermore, this is a rapidly evolving space, and we shouldn't be over-reactive. Fosstodon is itself open source, and we can play a part in helping them evolve their language/cultural openness stance (and I understand also that they are managing with very limited resources). Python and R, as mentioned above, also have a home there, and they are definitely very international communities. Rather than simply leaving fosstodon, we can help shape the conversation.

They also seem to offer tools for delegation/forwarding/linking accounts:

<img width="580" alt="image" src="https://user-images.githubusercontent.com/57394/208981485-d6b6749d-ff78-4a12-b8a2-30e271ce6baa.png">

In summary, my take is:

- We should not flag the fosstodon account as "the one and only official Jupyter home on Mastodon" yet. 
- We should keep it for now, as a placeholder whose role we can clarify in the coming weeks.
- We can explore further what options exist today, and what could change with further input in the future, to ensure better multi-lingual community support on Mastodon.

**Comment by SylvainCorlay** [comment]
Temporary things become permanent very quickly.

**Comment by SylvainCorlay** [comment]
I regret this decision. I think that temporary things become permanent very quickly. I advocated for removing this account as soon as possible because the longer it exists, the more people will feel like we have invested in it (we can already see the resistance right now, while it was only the effort of creating it).

Besides, I feel that the main reason for dismissing these concerns is that it is not considered as a real problem - as per the first reactions to the initial comment on other languages than English being banned from the platform. This definitely sends a strong signal that this is not a concern worth addressing.

It also unfortunate that after we stick to a strict formal decision making process for so long, we move forward with this as it happened almost by accident despite the concerns raised.

**Comment by SylvainCorlay** [comment]
To end this with a more positive note:
 - I am very optimistic about Mastodon, and I think it is a right move to have a presence in this network.
 - While I am unhappy with the pick of an anglocentric server, I understand the language question was off people's radar when picking Fosstodon, which otherwise looks like a reasonable choice.
 - I would be open to helping with the set up of a dedicated instance for Jupyter under a Jupyter domain.

**Comment by krassowski** [comment]
1) We could update the list of instances used by other open source projects as there were some new arrivals and some projects may have moved instances since I last compiled the list.
2) We could count which Mastodon servers are used by the followers of the Jupyter account. This is of course a biased metric, but might be somewhat informative in our search for a more suitable instance. Script and results with status as for today below, it's probably best to repeat once we get a bigger sample.

<details>

```python
import requests
import pandas as pd

headers = {'Accept': 'application/activity+json'}
data = requests.get('https://fosstodon.org/@Jupyter/followers', headers=headers).json()
next_page = data['first']
followers = []
while next_page:
    page = requests.get(next_page, headers=headers).json()
    followers.extend(page['orderedItems'])
    next_page = page.get('next')

print(
    pd.Series(followers)
    .str.extract('https://(.*)/users/')
    .value_counts()
    .to_markdown()
)
```


|                                     |   followers |
|:------------------------------------|----:|
| ('fosstodon.org',)                  |  45 |
| ('mastodon.social',)                |  15 |
| ('hachyderm.io',)                   |  13 |
| ('fediscience.org',)                |  11 |
| ('mast.hpc.social',)                |   8 |
| ('genomic.social',)                 |   5 |
| ('julialang.social',)               |   4 |
| ('mapstodon.space',)                |   3 |
| ('scholar.social',)                 |   3 |
| ('scicomm.xyz',)                    |   3 |
| ('piaille.fr',)                     |   3 |
| ('mastodon.online',)                |   3 |
| ('mstdn.science',)                  |   3 |
| ('mastodon.nz',)                    |   2 |
| ('openbiblio.social',)              |   2 |
| ('mstdn.social',)                   |   2 |
| ('mastodonapp.uk',)                 |   2 |
| ('mastodon.green',)                 |   2 |
| ('astrodon.social',)                |   2 |
| ('mas.to',)                         |   2 |
| ('mstdn.ca',)                       |   2 |
| ('sigmoid.social',)                 |   2 |
| ('social.coop',)                    |   2 |
| ('quinnwitz.house',)                |   1 |
| ('qubit-social.xyz',)               |   1 |
| ('post.lurk.org',)                  |   1 |
| ('sciencemastodon.com',)            |   1 |
| ('pdx.social',)                     |   1 |
| ('social.deckard.apps.glenux.net',) |   1 |
| ('social.juanlu.space',)            |   1 |
| ('octodon.social',)                 |   1 |
| ('social.linux.pizza',)             |   1 |
| ('social.ridetrans.it',)            |   1 |
| ('stoyko.space',)                   |   1 |
| ('mstdn.plus',)                     |   1 |
| ('mastodon.skrimmage.com',)         |   1 |
| ('mathstodon.xyz',)                 |   1 |
| ('kolektiva.social',)               |   1 |
| ('chaos.social',)                   |   1 |
| ('chat.rittmanchat.org',)           |   1 |
| ('data-folks.masto.host',)          |   1 |
| ('ecoevo.social',)                  |   1 |
| ('econtwitter.net',)                |   1 |
| ('framapiaf.org',)                  |   1 |
| ('fribygda.no',)                    |   1 |
| ('glammr.us',)                      |   1 |
| ('mamot.fr',)                       |   1 |
| ('mastodon.xyz',)                   |   1 |
| ('masto.ai',)                       |   1 |
| ('mastodon.cloud',)                 |   1 |
| ('mastodon.gamedev.place',)         |   1 |
| ('mastodon.mit.edu',)               |   1 |
| ('mastodon.nl',)                    |   1 |
| ('astronomy.city',)                 |   1 |
| ('mastodon.tetaneutral.net',)       |   1 |
| ('mastodon.top',)                   |   1 |
| ('vis.social',)                     |   1 |


</details>

**Comment by jasongrout** [comment]
@choldgraf - thanks for making the PR to revert the verified status of the account. I went ahead and approved and merged it.

My understanding from the discussion on Monday was that we were reserving the name, but not advertising it or making it official. As mentioned above, reserving the name was basically just preserving options for a later (but hopefully soon) discussion about our official social media strategy moving forward.

@SylvainCorlay - you're also on the NumFOCUS board. It seems that running a mastodon server is a great piece of infrastructure NumFOCUS could provide for projects to help build a strong community.

**Comment by jasongrout** [comment]
I see now that the Fosstodon account has our official logo, and certainly looks official. Regardless of whether it is "verified", I think it will be treated as official. As Sylvain says:

> I think that temporary things become permanent very quickly. I advocated for removing this account as soon as possible because the longer it exists, the more people will feel like we have invested in it

I think if we want to just have the name reserved for possible future use, we need to pull down the Jupyter logo and explicitly say in the description that it is a placeholder for potential future use. Or, as Sylvains says, delete it. Unlike Twitter, there is no central namespace for names, and we are the ones that decide what name/server is "official" Jupyter, so I don't feel like there is a huge advantage to jumping on a name before we have a strategy if it is confusing to do so.

**Comment by jasongrout** [comment]
To be more clear, I think we should make the [`@Jupyter` account ](https://fosstodon.org/@Jupyter) look like the [`@ProjectJupyter`](https://fosstodon.org/@ProjectJupyter) account, i.e., just a blank name with no official logo or official-looking text, at least until we decide on an official way forward.

**Comment by rkdarst** [comment]
I sent an email to fosstodon admins asking about multi-language, and got a reply saying that with mastodon 4 translation, it's possible and coming in a few days.  They would have changed the sooner, but the translation required an additional service to be run which they have been building over the last few days (currently testing).

(did no one else think to ask before all the panic?  Things have changed a lot over the last few months, I wouldn't be surprised if more policies need to change but haven't been updated yet!)

**Comment by SylvainCorlay** [comment]
> I sent an email to fosstodon admins asking about multi-language, and got a reply saying that with mastodon 4 translation, it's possible and coming in a few days. 

Wow, that is amazing news!

**Comment by jweill-aws** [comment]
@minrk @jasongrout Thanks for your responses while I was out on vacation Dec. 21–28. I have updated the Jupyter Fosstodon account to have a bio indicating that it's not in active use. I've also removed the banner and profile images, to avoid confusion for now. We should discuss our social strategy further. Of note, this rule of Fosstodon's suggests that we are expected to actively engage with people using the Jupyter account, not just post automatically, and we should monitor our `@`-mentions:

> Do not use automated tools to toot without participating in the community. It is fine to post with an automated tool so long as you are an active member of the Fosstodon community and respond to replies you receive here. Unmonitored accounts that post automatically are not acceptable.

(Note that "toot" is a term previously used by Mastodon to mean "post"; the latter term is now official)

**Comment by SylvainCorlay** [comment]
> I sent an email to fosstodon admins asking about multi-language, and got a reply saying that with mastodon 4 translation, it's possible and coming in a few days. They would have changed the sooner, but the translation required an additional service to be run which they have been building over the last few days (currently testing).

Do you know of any update or reference on this?

**Comment by rkdarst** [comment]
I don't have any update.  My only reference is private email but right after a server-wide announcement was made that mentions the translation service coming, so there is some independent reaction.

I'd recommend sending own email to ask if you are curios, or message the people in the announcement (`@kev@fosstodon.org` or `@mike@fosstodon.org`).  It would seem reasonable to even ask about what the expectation for organization accounts is... but I have very little personal interest here, the accounts work well for my organizations...

**Comment by SylvainCorlay** [comment]
Why not using hachyderm or a self-hosted instance just for Jupyter accounts?

**Comment by jasongrout** [comment]
I would love to have a numfocus instance, since that is the code of conduct and moderation body we've already agreed to and community we are a part of.

**Comment by SylvainCorlay** [comment]
> I would love to have a numfocus instance, since that is the code of conduct and moderation body we've already agreed to and community we are a part of.

Good point. Let's look at what's been going on in the infrastructure discussion.

**Comment by JasonWeill** [comment]
If we're still looking for a new home for Jupyter on Mastodon, Medium (which already hosts the Jupyter Blog) just launched its own Mastodon instance, with the same code of conduct as its blogging service: https://me.dm/

**Comment by JasonWeill** [comment]
While Fosstodon has still not relaxed its English-only rule, I would still like for us to have an official Jupyter Mastodon account, with all new microcontent posted to both Twitter and Mastodon, as a precaution against future Twitter outages and other challenges associated with the platform.

**Comment by minrk** [comment]
Should we go with the medium instance, then? Moving instances is very low impact, so we can move if/when we find a more comfortable home like a numfocus instance.

**Comment by blink1073** [comment]
Another option is [hachyderm](https://community.hachyderm.io/about/), which supports [Open Source Accounts](https://community.hachyderm.io/docs/account-types/open-source-accounts/), is multilingual friendly, and their stated values are prosocial.

**Comment by astrojuanlu** [comment]
Lots of projects set up shop on Hachyderm, it's a good choice.
/

**Comment by JasonWeill** [comment]
I agree that Hachyderm may be a good Mastodon instance for us. Their [sign-up page](https://hachyderm.io/auth/sign_up) has rules that seem to be in accord with our Code of Conduct, and I didn't find any restrictions that would dissuade Jupyter from being there.

**Comment by blink1073** [comment]
As of today, Fosstodon has lifted the restriction: "This rule was revoked as of August 8th, 2023 by a vote from our community.".   https://hub.fosstodon.org/why-fosstodon-is-english-only/

They also posted this announcement in the home feed:

"Following a discussion with our patrons, the policy requiring posts to be in English on Fosstodon has been removed. The moderation team will utilize available translation technologies for moderation purposes. However,
 we may request community support for languages not supported by our translation software, or where context isn't well understood. We appreciate your patience as we incorporate this change into our moderation practices."

**Comment by minrk** [comment]
hachyderm seems like a good choice right now.

**Comment by JasonWeill** [comment]
Any update on the creation of an official Mastodon account for Jupyter?

**Comment by blink1073** [comment]
Not as of yet, that will be among the first orders of business for the Jupyter Media Strategy Working Group, for which we are actively writing a charter.

**Comment by blink1073** [comment]
I have made a request for a Specialized Hachyderm account: https://github.com/hachyderm/community/issues/488

**Comment by blink1073** [comment]
We are live!  https://hachyderm.io/@ProjectJupyter

<img width="583" alt="image" src="https://github.com/jupyter/governance/assets/2096628/cd1433dc-d901-4cd7-aa3b-f5f550566f1e">

**Comment by JasonWeill** [comment]
Congrats! Blog post when? 😉

**Comment by astrojuanlu** [comment]
And would be nice to add a link at the bottom of jupyter.org :) 

![IMG_20231007_083227](https://github.com/jupyter/governance/assets/316517/2a1ee1bf-8eb9-4091-83ad-99c552417215)

**Comment by blink1073** [comment]
Done:  https://github.com/jupyter/jupyter.github.io/pull/742

**Comment by ivanov** [comment]
:broom: cleaning up here and closing this issue. 
Happy hacking! :bowtie:
